tokio-trace/src/lib.rs

```diff
@@ -475,15 +492,19 @@ macro_rules! callsite {
 /// ```
 #[macro_export]
 macro_rules! span {
-    ($name:expr) => { span!($name,) };
-    ($name:expr, $($k:ident $( = $val:expr )* ),*,) => {
-        span!($name, $($k $( = $val)* ),*)
+    (target: $target:expr, level: $lvl:expr, $name:expr, $($k:ident $( = $val:expr )* ),*,) => {
+        span!(target: $target, level: $lvl, $name, $($k $( = $val)*),*)
     };
-    ($name:expr, $($k:ident $( = $val:expr )* ),*) => {
+    (target: $target:expr, level: $lvl:expr, $name:expr, $($k:ident $( = $val:expr )* ),*) => {
         {
             use $crate::{callsite, field::{Value, ValueSet, AsField}, Span};
             use $crate::callsite::Callsite;
-            let callsite = callsite! { name: $name, fields: $( $k ),* };
+            let callsite = callsite! {
```

Comment:
I suspect that the `callsite! { name, fields }` case can now be removed from the `callsite!` macro (in `tokio-trace-core`) as I don't think it's used anywhere else.

Reply:
The only place we use that rule now is in tests: https://github.com/tokio-rs/tokio/blob/93e66e68ba6303f0f508896191492aaa77f16d98/tokio-trace/tests/support/field.rs#L202

I can change or remove the test if you would like, but it is still used.

Reply:
Okay, let's leave it as-is for now, then. We can always go back and remove it later.